### ISSUE TITLE
Sometimes we receive emojis as tofu even if displayed on android correctly

### ISSUE THREAD

**Issue by Jikstra** [deltachat/deltachat-desktop#527]
Hard to reproduce but with forwarding a message from android it works.

**Comment by Simon-Laux** [comment]
Idk how emojis are exactly handled at the moment (didn't looked to far into it yet)- maybe we should add https://www.google.com/get/noto/ as fallback
we could combine the noto emoji colorless and the mono font to not take up to much space and still support most languages and all emojis

**Comment by Jikstra** [comment]
@Simon-Laux not sure if it's really a font problem as the single emojis are rendered correctly, only in this arrangement (and definetly a lot others too) this happens. But I don't know much about this either.

**Comment by Simon-Laux** [comment]
https://stackoverflow.com/questions/11395584/fallback-fonts-on-special-characters

**Comment by Simon-Laux** [comment]
> @Simon-Laux not sure if it's really a font problem as the single emojis are rendered correctly, only in this arrangement (and definetly a lot others too) this happens. But I don't know much about this either.

You're right it's not the font - also because we render them currently as images and not as font

**Comment by hpk42** [comment]
Wrong Desktop Emoji displays now happens quite consistently.  On Android i see emojis correctly, on Desktop they look like the attachment. 
![emoji1](https://user-images.githubusercontent.com/73579/51842654-b2a3b480-2311-11e9-8e14-74165d00e690.png)

**Comment by Simon-Laux** [comment]
can you provide the emoji combinations trigger this bug? Am 28.01.2019 15:31 schrieb holger krekel <notifications@github.com>:Wrong Desktop Emoji displays now happens quite consistently.  On Android i see emojis correctly, on Desktop they look like the attachment.

**Comment by Jikstra** [comment]
Can we make sure that they're actually correct in the database? Not sure how to do that, but i think we should somehow get out the utf characters from the message. Not that we already have bullshit in the database.

**Comment by Simon-Laux** [comment]
The last state was that we blame it on `lib iconv`, right?

**Comment by nicodh** [comment]
Can't reproduce this. Does his still happen for anyone?

**Comment by Simon-Laux** [comment]
do you have the special combination that caused it? @nicodh

**Comment by nicodh** [comment]
I'm not sure what "special combination" mans. I forwarded a message with a couple of icons on android an and its displayed in desktop as expected. That's all I can see in this ticket...

Please send any special combination it needs to the UTF8 Testgruppe I just added in deltachat...

**Comment by Jikstra** [comment]
> I'm not sure what "special combination" mans. I forwarded a message with a couple of icons on android an and its displayed in desktop as expected. That's all I can see in this ticket...

It only happens with some special combinations, sadly i don't have that one anymore. Probably @hpk42 ? Otherwise maybe we need to kindly ask mattis for some emoji fuzzing tests :D

**Comment by nicodh** [comment]
Have you got my invitation to deltachat UTF8 Testgroup? There we have an example for that and discussed it...
